contrib/parseq-retry/src/main/java/com/linkedin/parseq/retry/RetryFailureException.java

```diff
@@ -0,0 +1,17 @@
+package com.linkedin.parseq.retry;
+
+/**
+ * Exception that is thrown when the process of retries is interrupted by any reason.
+ * Usually it's either the exhausted number of attempts, or the unacceptable task result.
+ */
+public class RetryFailureException extends Exception {
+  private static final long serialVersionUID = 0L;
+
+  public RetryFailureException() {
+    super();
+  }
+
+  public RetryFailureException(final String message) {
+    super(message);
+  }
```

Comment:
Yeah, I should also say that you can use the new constructor at line 160 in RetriableTask with the fatal error you get. That way when we look at the RetryFailureException from a failed parseq plan, we get see not only the error message but also the actual stack trace.